.buildkite/scripts/auth0-deployment.sh

```diff
@@ -19,14 +27,21 @@ function __do_deployment() {
   # Deploy database scripts
   for script in ${DATABASE_SCRIPTS}
   do
-    cp -v "${SCRIPTS_BUILD_DIR}/${script}.js" "${AUTH0_EXPORT_DIR}/database-connections/${AUTH0_CONNECTION_NAME}/"
+    dest_script="${AUTH0_EXPORT_DIR}/database-connections/${AUTH0_CONNECTION_NAME}/${script}.js"
+
+    cp -v "${SCRIPTS_BUILD_DIR}/${script}.js" "$dest_script"
+    echo -e "$DEBUG_HEADER\n\n$(cat "$dest_script")" > "$dest_script"
```

Comment:
let's use the webpack plugin that we already have for this; as well as being much more readable it means that commit hashes are associated with the build itself rather than the point at which the build is deployed (the latter could become misleading)